README.md

```diff
@@ -43,24 +54,179 @@ The plugin will remove the custom attribute from the element and replace its con
 You can use external libraries for this as well, no problem. Just make sure you are passing in a function that takes a string and returns a string. You might have to wrap the library function if it doesn't behave like this, but it will work with anything that transforms content.
 
 ```html
-<p md>Wow, it's **markdown**!</p>
+<p name>Transform me!</p>
 ```
 
 ```js
-const MarkdownIt = require('markdown-it')
-const md = new MarkdownIt() // can pass options here
-const content = require('posthtml-content')({
-  md: md.renderInline.bind(md)
+const fn = require('fn')
+
+const plugin = require('posthtml-content')({
+  name: (ctx) => fn(ctx)
+})
+
+posthtml([plugin]).process(html)
+```
+
+```js
+const fn = require('fn')
+
+const plugin = require('posthtml-content')({
+  name: fn.bind(fn)
+})
+
+posthtml([plugin]).process(html)
+```
+
+```html
+<p>👍</p>
```

Comment:
I would probably just add a function that does some sort of basic transformation, fully written out in the example, or even one that returns the contents but has a comment saying like "here's where you do your transformation"?